src/Digdir.Domain.Dialogporten.Infrastructure/UnitOfWork.cs

```diff
@@ -185,4 +201,20 @@ public void Dispose()
         _transaction?.Dispose();
         _transaction = null;
     }
+
+    // Although the entity framework supports all the options,
+    // enabling some of them ia an incredibly bad idea. Therefore,
+    // only some of them have setters until the day we actually
+    // have a use case for them.
```

Comment:
```suggestion
    // Although the Entity Framework supports all the options,
    // enabling some of them is an incredibly bad idea. Therefore,
    // only some of them have setters until the day we actually
    // have a use case for them.
```

Reply:
This doesn't sound very useful. Why are there options which is an incredibly bad idea to enable at all?